js_modules/dagit/packages/app/src/index.tsx

```diff
@@ -31,6 +32,7 @@ const config = {
   origin: process.env.REACT_APP_BACKEND_ORIGIN || document.location.origin,
   staticPathRoot: `${pathPrefix}/`,
   telemetryEnabled,
+  statusPolling: new Set<DeploymentStatusType>(['code-locations', 'daemons']),
```

Comment:
This looks good to me - I kinda wish cloud could just make the `daemons` status a no-op or quietly do nothing on the server-side so we didn't have to conditionally request the status, but this seems like a good client side approach!